aws/resource_aws_codebuild_project_test.go

```diff
@@ -2892,6 +2927,60 @@ resource "aws_codebuild_project" "test" {
 `, rName, authResource, authType)
 }
 
+func testAccAWSCodeBuildProjectConfig_Source_BuildStatusConfig(rName string) string {
+	return testAccAWSCodeBuildProjectConfig_Base_ServiceRole(rName) + fmt.Sprintf(`
```

Comment:
@bflad : Hi Brian, there is a terrafmt error here, but I am not sure how to rectify it, I tried running terrfmt locally and it didn't show any difference for this file. your thoughts ?